src/ws/login.js

```diff
@@ -397,6 +401,11 @@
         id("login-error-message").textContent = "";
     }
 
+    function clear_info() {
+        id("info-group").style.display = "none";
```

Comment:
Is setting `display=none` exactly the same as the `hidden` attribute? It feels a bit weird to use one in the html and the other in js

Reply:
I think that @KKoukiou was also solving this a few days ago - when `display: grid` is used then `hidden` in html has no effect. I think that I could just drop the `hidden` from html

Reply:
It's really much, much better to use a class, not inline CSS.

You'd want to use `display: none` in the class. The hidden attribute is not supported so well, as CSS can override it if you're not careful.

See:
- https://meowni.ca/hidden.is.a.lie.html
- https://css-tricks.com/the-hidden-attribute-is-visibly-weak/

> when display: grid is used then hidden in html has no effect

That's not quite accurate. See the above links. :wink:

Anyway, don't use JavaScript to fiddle with CSS; use it to add/remove classes which are used in CSS.